regression/contracts-dfcc/invar_loop-entry_check/main.c

```diff
@@ -12,7 +12,7 @@ void main()
   x1 = &z1;
 
   while(y1 > 0)
-    __CPROVER_loop_invariant(*x1 == __CPROVER_loop_entry(*x1))
+    __CPROVER_loop_invariant(y1 >= 0 && *x1 == __CPROVER_loop_entry(*x1))
```

Comment:
Why is this (and the other changes in this file) now necessary?

Reply:
It is for the same reason why we add `nondet_int`. It is safe to ignore `y1` in DFCC instrumentation. But we want to trigger the checks for `y1` as described in the `.desc` file. Adding a new invariant clause will trigger all DFCC checks for `y1`.